src/app.js

```diff
@@ -197,11 +199,14 @@ export function app(app) {
     function removeChild() {
       parent.removeChild(element)
     }
+    recoverNode(node)
```

Comment:
This is why I started the PR to ask if this was the correct place to hook into recycle.  I really don't know if it's wrong or right, but all of our tests still pass.